README.md

```diff
@@ -14,30 +14,256 @@ At the moment, this SDK includes the following features:
 - [x] Marshalling & unmarshalling of [JSON-Cadence](https://docs.onflow.org/cadence/json-cadence-spec/)
 - [x] DSL for creating, signing, and sending transactions and scripts
 
-## Repository modules
+## Table of Contents
+- [Repository structure](#repository-structure)
+  - [SDK](#sdk)
+  - [Java Example](#java-example)
+  - [Kotlin Example](#kotlin-example)
+  - [Common Utils](#common-utils)
+- [Contribute to this SDK](#contribute-to-this-sdk)
+- [Dependencies](#dependencies)
+- [Getting Started](#getting-started)
+  - [Installation](#installation)
+  - [Generating Keys](#generating-keys)
+  - [Supported Curves](#supported-curves)
+  - [Accessing the Flow Network](#accessing-the-flow-network)
+  - [Creating an Account](#creating-an-account)
+  - [Signing Transactions](#signing-transactions)
+  - [Sending Transactions](#sending-transactions)
+  - [Querying Transaction Results](#querying-transaction-results)
+  - [Querying Blocks](#querying-blocks)
+  - [Executing a Script](#executing-a-script)
+  - [Querying Events](#querying-events)
+  - [Querying Accounts](#querying-accounts)
+  - [Examples Summary](#examples-summary)
+- [Credit](#credit)
+
```

Comment:
Let's add a section below called 'Unsupported features'. We can summarize the missing support for Websockets and Event streaming APIs for now.